cdc/model/changefeed.go

```diff
@@ -259,12 +260,16 @@ func (info *ChangeFeedInfo) VerifyAndComplete() error {
 
 // FixIncompatible fixes incompatible changefeed meta info.
 func (info *ChangeFeedInfo) FixIncompatible() {
+	log.Info("Start fixing incompatible changefeed info")
```

Comment:
Does it need to log changefeed info before fixing and after?

Reply:
Yes, but several fix functions actually output what it fixes. it may output some sensitive information.

Reply:
all right, may need some function to hide sensitive info of changefeed info.

Reply:
Addressed in https://github.com/pingcap/tiflow/pull/3895/commits/971aace693a84e93e801fe93aa86f7e18918ffe4